src/api/views/template_views.py

```diff
@@ -133,16 +134,18 @@ class TemplateAttributesView(MethodView):
 
     def get(self):
         """Get list of keys for template context."""
+        fake = Faker()
+
         return jsonify(
-            [
-                "address",
-                "city",
-                "description",
-                "email",
-                "name",
-                "phone",
-                "state",
-            ]
+            {
+                "address": fake.street_address(),
+                "city": fake.city(),
+                "description": "",
+                "email": "",
+                "name": "",
```

Comment:
So I'm approving the PR, but I'm holding on the merge. I'm wondering why you didn't use fake.email() and fake.name() for these two?

Reply:
because the email and name should match up with the domain name